src/PIL/FtexImagePlugin.py

```diff
@@ -94,7 +94,8 @@ class FtexImageFile(ImageFile.ImageFile):
     format_description = "Texture File Format (IW2:EOC)"
 
     def _open(self):
-        struct.unpack("<I", self.fp.read(4))  # magic
+        if not _accept(self.fp.read(4)):
+            raise SyntaxError("not a FTEX file")
```

Comment:
Nit: be guided by pronunciation:
```suggestion
            raise SyntaxError("not an FTEX file")
```